R/response_sources.R

```diff
@@ -0,0 +1,259 @@
+#' Pre-Defined Event Objects
```

Comment:
```suggestion
#' Pre-Defined Response Event Objects
```
Matches the new name better?

Reply:
Yes, updated